turbopack/crates/turbopack-ecmascript/tests/tree-shaker/analyzer/route-handler/output.md

```diff
@@ -147,20 +147,13 @@ export { };
 ```
 ## Part 1
 ```js
-import "__TURBOPACK_PART__" assert {
-    __turbopack_part__: 0
-};
-import { NextResponse } from "next/server";
-export { NextResponse as b } from "__TURBOPACK_VAR__" assert {
-    __turbopack_var__: true
-};
 
 ```
```

Comment:
This part is empty now, is that expected? Can we remove it entirely then?

Reply:
It's expected, but it's still required to create a unique item id.
I can remove them from the test output, though.

Reply:
> I can remove them from the test output, though.

(If we do need to keep it, leave it in the output, so that you can see what's going on.)